CHANGELOG.md

```diff
@@ -40,6 +40,12 @@ Main (unreleased)
 
 - `pyroscope.ebpf` support python on arm64 platforms. (@korniltsev)
 
+- `otelcol.receiver.prometheus` does not drop histograms without buckets anymore. (@wildum)
+
+- The labels`otel_scope_name` and `otel_scope_version` in `otelcol.receiver.prometheus` are dropped and used as Instrumentation Scope name and version respectively. All `otel_scope_info` metrics are dropped and labels on `otel_scope_info` metric points other than `otel_scope_name` and `otel_scope_version` are added as scope attributes with the matching name and version. (@wildum)
```

Comment:
I think this should go in a "Breaking changes" section, [after](https://github.com/grafana/agent/blob/main/docs/developer/contributing.md#updating-the-changelog) the "Security fixes" section. 
However, although this is a breaking change, I don't think it will be a problem for users. I doubt that anyone would mind if their OTLP metrics have proper instrumentation scope being set.